core/src/main/java/org/apache/iceberg/rest/responses/LoadTableResponse.java

```diff
@@ -89,8 +144,15 @@ public static class Builder {
     private Builder() {
     }
 
+      public Builder withMetadataLocation(String metadataLocation) {
+          this.metadataLocation = metadataLocation;
+          return this;
+      }
```

Comment:
+1 to this. I find that sometimes, the `TableMetadata` that gets passed has `null` for `metadataLocation`, in which case we need to use the `TableMetadataBuilder` to add it which isn't very convenient.  👍